lists/linked_list.py

```diff
@@ -0,0 +1,72 @@
+class Node:
+
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+
+
+class LinkedList:
+
+    def __init__(self):
+        self._head = None
+        self._count = 0
+
+    def add(self, item):
+        node = Node(item)
+        if self._head is None:
+            self._head = node
+        else:
+            p = self._head
+            while p.next is not None:
+                p = p.next
+            p.next = node
+        self._count += 1
+
+    def remove_head(self):
+        item = self._head.data
+        next = self._head.next
```

Comment:
yo, mario, avoid naming a variable after the builtins functions. Add an underscore like you did at line 61.